src/Fmt_ast.ml

```diff
@@ -4072,10 +4065,18 @@ and fmt_let_op c ctx ~ext ~parens ~fmt_atrs ~fmt_expr bindings
         | `Sparse -> "@;<1000 0>"
         | `Compact -> "@ " )
   in
+  let blank_line_after_in =
+    let last_bind = List.last_exn bindings in
+    (* The location of the first binding (just after `let`) is wrong, it
+       contains the whole letop expression *)
+    let last_bind_expr_loc = last_bind.pbop_exp.pexp_loc in
```

Comment:
@gpetiot There was something wrong with the locations. The location of `body` in `Pexp_letop {let_; ands; body}` is right. The problem was that the location of `let_` contains the whole expression, overlapping with `ands` and `body`.